applications/app/controllers/InteractiveController.scala

```diff
@@ -128,6 +130,12 @@ class InteractiveController(
 
     if (isUSElectionAMP) { // A special-cased AMP page for various US Election (2020) interactive pages.
       renderUSElectionAMPPage(path)
+    } else if (ActiveExperiments.isParticipating(InteractiveLibrarianExp)) {
+      val result = InteractiveLibrarian.getDocumentFromS3(path) match {
+        case Some(document) => Ok(document).as("text/plain")
```

Comment:
Should the MIME type be `text/html`? 

Are cache headers being set elsewhere, or should they be added here?

Reply:
I should have commented that one. I set it as "text/plain" during development to make in unambiguous that I am getting the file from S3 instead of the regular document generated by the backend. Once I am convinced that the correct files are being read I will give it the expected MIME type :)